src/quick-lint-js/error.h

```diff
@@ -572,9 +572,10 @@
                                                                                \
   QLJS_ERROR_TYPE(                                                             \
       error_missing_body_for_if_statement, "E064",                             \
-      { source_code_span if_and_condition; },                                  \
+      { source_code_span after_if_condition; },                                \
       .error(QLJS_TRANSLATABLE("missing body for 'if' statement"),             \
-             if_and_condition))                                                \
+             source_code_span(after_if_condition.end(),                        \
+                              after_if_condition.end())))                      \
```

Comment:
I think it's weird to make `after_if_condition` an empty span (in parse.h), then unpack it and recreate the span here.

Either use `after_if_condition` as-is here, or leave `after_if_condition` as it was before in parse.h.

Reply:
Are you saying a change like this?  

` QLJS_ERROR_TYPE(                                                             
      error_missing_body_for_if_statement, "E064",                             
      { source_code_span expected_body; },                                     
      .error(QLJS_TRANSLATABLE("missing body for 'if' statement"),             
             expected_body))       
`

`const char8 *end_of_if_condition = this->lexer_.end_of_previous_token();
      this->error_reporter_->report(error_missing_body_for_if_statement{
          .expected_body =
              source_code_span(end_of_if_condition, end_of_if_condition),`